ckanext/ontario_theme/schemas/ontario_theme_organization.json

```diff
@@ -30,6 +30,16 @@
           "preset": "fluent_core_translated",
           "classes": ["col-md-12","form-group"]
         },
+        {
+          "field_name": "org_contact_translated",
+          "label": {
+            "en": "Organization Contact",
+            "fr": "FR_Organization Contact"
+          },
+          "preset": "fluent_core_translated",
```

Comment:
can you change this type to "fluent_text"? fluent_core_translated is just for ckan core fields